src/components/TextArea/__snapshots__/TextArea.test.tsx.snap

```diff
@@ -6,7 +6,8 @@ exports[`<TextArea /> with all options matches its snapshot 1`] = `
   ariaLabel="ARIA_LABEL"
   autoAdjustHeight={true}
   borderless={false}
-  className="y-base-text-field y-text-area CLASS"
+  className="y-textField y-textArea CLASS"
```

Comment:
Does `y-textField` have the base text field styles? Should it have `base` in the name?

Reply:
the "TextField" class is the internal component. after this i'll follow up with a PR to rename the public TextField component to TextInput for a bit more clarity
https://www.yammer.com/microsoft.com/#/Threads/show?threadId=1085910906